src/artemis/external_interaction/tests/test_ispyb_handler.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import MagicMock, call
+
+from artemis.external_interaction.communicator_callbacks import ISPyBHandlerCallback
+from artemis.parameters import FullParameters
+
+
+def test_fgs_failing_results_in_bad_run_status_in_ispyb(
+    mock_ispyb_update_time_and_status: MagicMock,
+    mock_ispyb_get_time: MagicMock,
+    mock_ispyb_store_grid_scan: MagicMock,
+    wait_for_result: MagicMock,
+    run_end: MagicMock,
+    run_start: MagicMock,
+    nexus_writer: MagicMock,
+    td,
```

Comment:
Nit: Can you type hint this?

Reply:
no, I can't import the type without adding an init file and making the test folder a module